src/Packetery/Module/Order/GridExtender.php

```diff
@@ -135,6 +135,13 @@ public function addFilterLinks( array $var ): array {
 	 * Adds select to order grid.
 	 */
 	public function renderOrderTypeSelect(): void {
+		global $pagenow, $typenow;
+
+		$isOrderGridPage = ( 'edit.php' === $pagenow && 'shop_order' === $typenow );
```

Comment:
isOrderGridPage - máme už v enqueueAdminAssets().  Navrhuji vyčlenit podmínku , která vrací informaci zda se jedná o seznam objednávek do vlastní metody, kterou použije na všech místech.  Volání této metody, pak použiješ v podmínce o řádek níže